iOS/Flows/Conversation/Conversation Header/MemberCell.swift

```diff
@@ -12,14 +12,19 @@ import Lottie
 
 struct Member: Hashable, Equatable {
     var displayable: AnyHashableDisplayable
-    var isTyping: Bool
+    var conversationController: ConversationController
```

Comment:
I realized an isTyping bool is quite the approach because it changes the hash of the item. That means it treats flipping the bool like creating an entirely new item.

A better approach is to a reference to the data, in this case the controller.